### PR TITLE
GEO-45: Make geo_shape configurable in pelias-schema 

### PR DESCRIPTION
**Pull request by ainemitch**
…index - initial commit.

### CODE REVIEW

File: scripts/create_index.js [hunk 1]
```diff
@@ -6,6 +6,15 @@ const SUPPORTED_ES_VERSIONS = '>=7.4.2';
 const cli = require('./cli');
 const schema = require('../schema');
 
+// Check that the shape_type property has been defined in the schema.js file
+// This must be set to either 'shape' or 'polygon', if not exit
+if(shapeType === 'shape' || shapeType === 'polygon'){
+  console.log('\n Creating an ES index with schema shape type: ' + shapeType + '.');
```
Comment (alenkalac) [inline scripts/create_index.js]: Hi Aine, you might be able to use template literal here with backticks, that way you can avoid string concat and might look a bit cleaner, up to you though. 
```
  console.log(`\n Creating an ES index with schema shape type: ${shapeType}.`);
```

Comment (ainemitch) [reply scripts/create_index.js]: Cheers Alen I updated that there now.

**Review by tomislavkosutic — APPROVED** [review]
Thanks Aine.

**Review by alenkalac — APPROVED** [review]
Thumbs up!

**Pull request merged**